app/Http/Controllers/Account/AcceptanceController.php

```diff
@@ -119,6 +124,21 @@ public function store(Request $request, $id)
             $return_msg = trans('admin/users/message.declined');
         }
 
+        $item = $acceptance->checkoutable_type::find($acceptance->checkoutable_id);
+
+        $data = [
+            'item' => $item,
+            'eula' => $item->getEula(),
+            'signature' => storage_path().'/private_uploads/signatures/'.$sig_filename,
+            'logo' => public_path().'/uploads/snipe-logo.png',
+        ];
+
+        \Log::error(storage_path().'/eula-pdfs/'.$sig_filename);
+
+        $pdf = Pdf::loadView('account.accept-eula', $data);
+        Storage::put('private_uploads/eula-pdfs/accepted-eula-'.date('Y-m-d-h-i-s').'.pdf', $pdf->output());
```

Comment:
`loadView()` just tells it what blade view want to use for this PDF. It's possible to use `loadHtml()` instead, but it's super clunky. Using a blade here makes it a lot easier to tweak the designs.